[2024-02-29T06:07:37.663+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data.call_api_and_save_response__1 manual__2024-02-29T06:07:36.645032+00:00 [queued]>
[2024-02-29T06:07:37.671+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data.call_api_and_save_response__1 manual__2024-02-29T06:07:36.645032+00:00 [queued]>
[2024-02-29T06:07:37.671+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-02-29T06:07:37.694+0000] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): call_api_and_save_response__1> on 2024-02-29 06:07:36.645032+00:00
[2024-02-29T06:07:37.722+0000] {standard_task_runner.py:60} INFO - Started process 77 to run task
[2024-02-29T06:07:37.729+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data', 'call_api_and_save_response__1', 'manual__2024-02-29T06:07:36.645032+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/fetch_data.py', '--cfg-path', '/tmp/tmph5sv9da5']
[2024-02-29T06:07:37.739+0000] {standard_task_runner.py:88} INFO - Job 29: Subtask call_api_and_save_response__1
[2024-02-29T06:07:37.805+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data.call_api_and_save_response__1 manual__2024-02-29T06:07:36.645032+00:00 [running]> on host 0a26efe0197d
[2024-02-29T06:07:37.929+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='root' AIRFLOW_CTX_DAG_ID='fetch_data' AIRFLOW_CTX_TASK_ID='call_api_and_save_response__1' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T06:07:36.645032+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-29T06:07:36.645032+00:00'
[2024-02-29T06:07:38.574+0000] {logging_mixin.py:188} INFO - API response saved successfully to /opt/***/data/irctc_data.json
[2024-02-29T06:07:38.577+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-02-29T06:07:38.590+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data, task_id=call_api_and_save_response__1, execution_date=20240229T060736, start_date=20240229T060737, end_date=20240229T060738
[2024-02-29T06:07:38.628+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-29T06:07:38.643+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
